addons/io_scene_gltf2/blender/imp/gltf2_blender_gltf.py

```diff
@@ -178,6 +178,10 @@ def pre_compute(gltf):
                     if 'glossinessFactor' not in material.extensions['KHR_materials_pbrSpecularGlossiness'].keys():
                         material.extensions['KHR_materials_pbrSpecularGlossiness']['glossinessFactor'] = 1.0
 
+        # images
+        for img in gltf.data.images:
```

Comment:
> Error: Command failed: blender279b -b --addons io_scene_gltf2 -noaudio --python roundtrip_gltf.py -- roundtrip/01_color_attribute/01_color_attribute.gltf /out/roundtrip/01_color_attribute/outblender279b 
2019-04-16

> File "gltf2_blender_gltf.py", line 182, in pre_compute for img in gltf.data.images:
> TypeError: 'NoneType' object is not iterable

Reply:
I committed a fix in the roundtrip_ao_node branch